lib/components/Box/useBoxStyles.treat.ts

```diff
@@ -370,3 +370,9 @@ export const relativePosition = {
     mapToStyleProperty(relativePositionRules, 'right'),
   ) as PositionRulesType,
 };
+
+export const userSelect = styleMap({
+  auto: { userSelect: 'auto' },
```

Comment:
Since this is the default value, this shouldn't be required, right?

Reply:
Good catch, will remove it :)